### ISSUE TITLE
Explore Improving Logging of A11y Sitemap Tests to Jenkins

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#22838]
## Research Story
Is it possible to improve the logging of the Nightwatch A11y Sitemap tests into Jenkins for easier consumption of success/fail of individual a11y evaluations?

## Expected Behavior
The log output from the Nightwatch Sitemap Accessibility tests are presented with greater detail in Jenkins so as to allow for better troubleshooting of issues.

## Acceptance Criteria

- [ ] Nightwatch Sitemap A11y Tests are easily readable in Jenkins
- [ ] Test failures log details that help users diagnose the issue to Jenkins
- [ ] Test passes log only essential details to Jenkins

**Comment by aubreyarcangel** [comment]
After speaking to @1Copenut it was determined the work he's doing in [this ticket](https://github.com/department-of-veterans-affairs/vets-website/pull/16424/) improves the logging of the sitemap tests in Jenkins

https://github.com/department-of-veterans-affairs/vets-website/pull/16424/files#diff-8ed007b90f7ef4792d4f052faca231f5c2e2297c38c1ecb9d8e4155fb621aaabR77

**Comment by pjhill** [comment]
Duplicative of Trevor's work elsewhere -- closing
